typescript-arrays/main.ts

```diff
@@ -0,0 +1,23 @@
+const colors: any = ['red', 'white', 'blue'];
```

Comment:
You want to avoid using the `any` type, because it provides no type checking or safety. If the only things inside of this array are going to be strings, then it should be typed as an array of strings. You can review the `typescript-jumpstart` guide or `typescript-intro` to review syntax on that.